playground/src/PlaygroundView.tsx

```diff
@@ -69,14 +71,16 @@ export class PlaygroundView extends React.Component<IPlaygroundViewProps, IPlayg
 
   private _renderInputBox(): React.ReactNode {
     return (
-      <FlexColDiv className='playground-input-box' style={ { flex: 1 } }>
+      <FlexColDiv className='playground-input-box' style={ { flex: 1, width: '50%' } }>
         <div style={ { height: '40px' } } />
-        <textarea
+        <MonacoWrapper
           className='playground-input-textarea'
           style={ { width: '100%', height: '100%', boxSizing: 'border-box', resize: 'none' } }
           value={ this.state.inputText }
           onChange={ this._inputTextArea_onChange.bind(this) }
-          />
+          language='typescript'
+         />
+        {/*  */}
```

Comment:
>{/*  */} [](start = 8, length = 8)

?

Reply:
Oops - removed.

---
In reply to: [223514820](https://github.com/Microsoft/tsdoc/pull/90#discussion_r223514820) [](ancestors = 223514820)